### ISSUE TITLE
Input plug: option to convert result to a single array attribute

### ISSUE THREAD

**Issue by nmaguiar** [OpenAF/nAttrMon#85]
To decrease the number of attributes in some nAttrMon configurations it would be helpful to convert an input plug result consisting of a set of attributes (e.g. endpoints) into a single attribute where the previous attribute names become a key:

Input plug result example:
````javascript
attributes: {
   attr1: {
      result: true,
      errorMessage: "n/a"
   },
   attr2: {
      result: false,
      errorMessage: "some error"
   }
}
```` 

New plug result example after applying the new parameter:
````javascript
attributes: {
   newAttr1: [
      { key: "attr1", result: true, errorMessage: "n/a" },
      { key: "attr2", result: false, errorMessage: "some error" }
   ]
}
````

Note: this should be done before the merge plug option so it can also be later merged into a single attribute from other plug inputs.
